optimize/components/userguide/additional-features/data-source-variants.md

```diff
@@ -0,0 +1,17 @@
+---
+id: data-source-variants
+title: Data source variants
```

Comment:
I suggest calling this `process variants comparison` because It is much more intuitive to the user to understand.

We also have a [plan](https://jira.camunda.com/browse/OPT-5584) to completely get rid of the `data source` term.